### ISSUE TITLE
A11y_SQLMigrationControlPlane_MigrateToAzureSQL_ DatabasesForAssessment_ScreenReader: Column header for the table is not announced by the screen reader.

### ISSUE THREAD

**Issue by BhanuPrakashGit** [microsoft/azuredatastudio#20626]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11yMAS;#A11yTCS;#AzureDataStudioextensionAzureSQLMigration-Mac-Sept2022;#SQL Migration Control Plane;#A11ySev2;#WCAG1.3.1;#DesktopApp;#Win11;#ScreenReader;#Benchmark;#VoiceOver;#Mac

### Environment Details:
Azure Data Studio: Version: 1.39.1
Operating System: Mac Version 12.5.1
Screen Reader: VoiceOver

### Steps to Reproduce:
1. Install Azure data studio and open
2. Tab till Extension press enter key and install Azure SQL Migration 
3. Tab till Respective server and right click
4. Tab till manage and press enter key
5. Tab till Azure SQL Migration and press enter key
6. Tab till migrate to azure SQL and press enter, saved session page will open 
7. Select start new session and give next, Databases for assessment page will open 
8. Open screen reader and observe that column header is not announced by the voice over.

### Actual:
 Column header for the table is not announced by the screen reader. It is only announced the name present in the column.

### Expected:
 Column header for the table should be announced by the screen reader. It should announce like Database column header, etc. 

### User Impact:
Screen reader user will get understand the column header for the table.
[Screen Recording 2022-09-14 at 1.14.00 PM.zip](https://github.com/microsoft/azuredatastudio/files/9606900/Screen.Recording.2022-09-14.at.1.14.00.PM.zip)

**Comment by InduPriya1805** [comment]
Githubtags:  #AzureDataStudioextensionAzureSQLMigration-Mac-Sept2022;

**Comment by aasimkhan30** [comment]
@BhanuPrakashGit and @InduPriya1805 
This seems to be a voice over issue. Here is an example demonstrating that voiceover does not read column headers of table. 
https://codepen.io/aasimkhan30/pen/yLxexmL

**Comment by BhanuPrakashGit** [comment]
@aasimkhan30 Logged IRT bug, External to VoiceOver.

**Comment by brian-harris** [comment]
@BhanuPrakashGit since this is an external issue, can the severity of this be bug dropped from 2 to 3, as it is external and impacting our SLA.

**Comment by brian-harris** [comment]
@BhanuPrakashGit this issue is impacting our SLA, and we do not have the ability to fix it.  I would like to drop the bug severity from 2 to 3 so we are not graded for external issues.

**Comment by BhanuPrakashGit** [comment]
The external bug is closed as By Design. As per the discussion with SME, closing this bug.
GithubTags:#Regressed:09-27-23;#Closed;
